main.py

```diff
@@ -50,15 +63,23 @@ def menu():
 def main():
     count_attemps = 1
     login_success = False
+    correct_format = False
     while count_attemps <= 3:
-        user_id = get_number_from_input("Enter your ID: ")
-        user_pass = get_number_from_input("Enter you password: ")
-        login_success = check_id_and_pin(user_id, user_pass)
-        if not login_success:
-            print(f"Something wrong with your ID or PASS. {count_attemps}/3. Try again")  # need not to TRY AGAIN at the last time
-            count_attemps += 1
+        user_id = input("Enter your ID: ")
```

Comment:
think about how you can write it better.
too messy code. Some of action you may do more clear